File: src/main/java/com/ject/studytrip/trip/domain/repository/TripReportQueryRepository.java
package com.ject.studytrip.trip.domain.repository;

import com.ject.studytrip.trip.domain.model.TripReport;
import java.util.List;

public interface TripReportQueryRepository {
    List<TripReport> findAllActiveByMemberId(Long memberId);

    List<String> findImageUrlsByMemberId(Long memberId);
}

File: src/main/kotlin/com/ject/studytrip/trip/domain/repository/TripReportQueryRepository.kt
package com.ject.studytrip.trip.domain.repository

import com.ject.studytrip.trip.domain.model.TripReport

interface TripReportQueryRepository {
    fun findAllActiveByMemberId(memberId: Long): List<TripReport>

    fun findImageUrlsByMemberId(memberId: Long): List<String>
}
